prereise/gather/winddata/impute.py

```diff
@@ -137,3 +137,26 @@ def gaussian(data, wind_farm, inplace=True, curve="state"):
 
     if not inplace:
         return data_impute
+
+
+def linear(data, inplace=True):
+    """Given a 2D array, linearly interpolate any missing values column-wise.
+
+    :param numpy.array data: data to interpolate.
+    :param bool inplace: whether to modify the data inplace or return a modified copy.
+    :return: (*None/pandas.DataFrame*) -- if ``inplace`` is False, data frame with
+        missing entries imputed.
+    """
+    data_impute = data if inplace else data.copy()
+    # Iterate through columns
+    for i, a in enumerate(data_impute.T):
+        a_nan = np.isnan(a)
+        if a_nan.sum() == 0:
+            continue
+        data_impute[a_nan, i] = np.interp(
+            a_nan.nonzero()[0],  # Indices of bad data
+            (~a_nan).nonzero()[0],  # Indices of good data
+            a[~a_nan],  # Values of good data
+        )
+    if not inplace:
+        return data_impute
```

Comment:
I'm wondering whether we could use `pandas.DataFrame.interpolate(method="linear")` which can be done directly over the resultant data frame with `np.nan`?

Reply:
I think you are right!

Reply:
That said it would be nice to have a `linear` function in the `impute` module so perhaps we can have a wrapper around `pandas.DataFrame.interpolate(method="linear")`

Reply:
Done.